### DISCUSSION TITLE
Expanded or floating TOC in the main window?

### DISCUSSION THREAD

**Discussion by thehoegh** [McShelby/hugo-theme-relearn#754] [Q&A] [answered]
Is there a way to achieve the TOC in the main window to always be expanded, or maybe even floating at a given position in the main window? 

![image](https://github.com/McShelby/hugo-theme-relearn/assets/4199918/5e2f8d53-8cae-4c10-8a3a-10272e4b0e25)

Thanks a great theme, I really appreciate the work 😊

**Comment by McShelby** [comment, marked as answer]
At the moment the toc can only be displayed that way.

I have some plans to add more options for the position of the toc as I am not happy myself. Sadly, this is very low on my todo list, so it may take a while till I implement something in this area.

**Reply by thehoegh** [reply]
Thanks for the swift reply 😊
